### PR TITLE
CI: Run browser, node and fastboot tests in parallel

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

**Comment by marcoow** [comment]
Yeah, getting build times back to a decent level would be a big relief ❤️

**Comment by marcoow** [comment]
Strange thing is this runs just as long as the non-parallel builds: https://travis-ci.org/simplabs/ember-simple-auth/jobs/234047753

😕

**Comment by Turbo87** [comment]
yeah... I'm a little confused by that too 🤔

**Comment by Turbo87** [comment]
FWIW I still consider this a win since we can see the log output of the node and fastboot tests again 😉

**Comment by marcoow** [comment]
Absolutely - we should investigate the time thing though…

File: .travis.yml [hunk 1]
```diff
@@ -22,11 +28,9 @@ install:
...
 
 script:
-  # Usually, it's ok to finish the test scenario without reverting
-  #  to the addon's original dependency state, skipping "cleanup".
-  - npm run test:all -- --skip-cleanup
-  - npm run test:node
-  - npm run test:fastboot
+  - if [[ $TESTS == 'browser' ]]; then npm run test:all; fi
+  - if [[ $TESTS == 'node' ]]; then npm run test:node; fi
+  - if [[ $TESTS == 'fastboot' ]]; then npm run test:fastboot; fi
```
Comment (Turbo87) [inline .travis.yml]: @marcoow updated

**Comment by marcoow** [comment]
Argh, build is failing due to updated FastBoot.

**Comment by marcoow** [comment]
This needs to be rebased on `master` once #1367 is merged.

**Pull request merged**